presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -179,7 +179,10 @@ Property Name                                      Description
 ``hive.collect-column-statistics-on-write``        Enables automatic column level statistics collection         ``false``
                                                    on write. See `Table Statistics <#table-statistics>`__ for
                                                    details.
-================================================== ============================================================ ============
+``hive.s3select-pushdown.enabled``                 Enable query pushdown to AWS S3Select service.               ``false``
```

Comment:
It may be worth adding some of these info to the doc. I expect that many users/cluster admins would be interested in knowing the implications of enabling this config in terms of performance, cost, etc.

Reply:
Sure will do.

Reply:
Updated documentation in ``Is S3 Select Right For My Application?`` section, line ``356``.